### DISCUSSION TITLE
Adding point light to scene

### DISCUSSION THREAD

**Discussion by Jureq1234** [isl-org/Open3D#4389] [Q&A]
Hi all,

I'm trying to add point light to the scene and render it. 
but it seems like the light is not added :(.
Where did I make a mistake?
`
 box = o3d.geometry.TriangleMesh.create_box(width=1, height=1, depth=1).translate([0, 0, 1]).subdivide_midpoint(
        3)
    box2 = o3d.geometry.TriangleMesh.create_box(width=100, height=100, depth=0.1).translate([0, 0, 1]).subdivide_midpoint(
        3)
    box3 = o3d.geometry.TriangleMesh.create_box(width=.1, height=.1, depth=0.1).translate(
        [1, 1, 4]).subdivide_midpoint(
        3)
    box.compute_vertex_normals()
    box.paint_uniform_color([0.1, 0.1, 0.7])
    box2.compute_vertex_normals()
    box2.paint_uniform_color([0.1, 0.1, 0.7])
    box3.compute_vertex_normals()
    box3.paint_uniform_color([0.1, 0.1, 0.7])

...

    render = o3d.visualization.rendering.OffscreenRenderer(w, h)
    render.scene.scene.enable_indirect_light(False)
    render.scene.add_geometry("box", box, material)
    render.scene.add_geometry("box2", box2, material)
    render.scene.add_geometry("box3", box3, material)
    render.scene.scene.enable_sun_light(False)
    render.scene.show_axes(False)
    render.scene.scene.add_point_light('light', [1, 1, 1], [1, 1, 3], 10000e6, 100e2, True)
    render.scene.scene.add_spot_light('light2', [1, 1, 1], [1, 1, 3], [1, 1, -1], 200000e6, 200e2, 20, 22, True)`

![image](https://user-images.githubusercontent.com/88145971/145099188-bf1ccb05-fc97-4d42-8e69-5d8d56be01b4.png)
)

**Comment by errissa** [comment]
Please take a look at the example in #3970.

**Reply by Jureq1234** [reply]
I have checked it, but it doesn't work with rendering.OffscreenRenderer .
